### ISSUE TITLE
vminsert: cannot parse accountID from "multitenant"

### ISSUE THREAD

**Issue by chevvi** [VictoriaMetrics/VictoriaMetrics#4947]
### Describe the bug

vminsert is warning and doesn't accept metrics, which send into multitenant because it doesn't understand what 'multitenant' is.

vmagent is configured to send metrics to multitenant (Multitenancy via labels). 
`-remoteWrite.url=http://<vminsert>:8480/insert/multitenant/prometheus/api/v1/write`
Label is added
```
    relabel_configs:
      - target_label: "vm_account_id"
        replacement: "2"
```

But vminsert get warn
`warn VictoriaMetrics/app/vminsert/main.go:213 remoteAddr: "10.10.1.161:55742"; requestURI: /insert/multitenant/prometheus/api/v1/write; auth error: cannot parse accountID from "multitenant": strconv.ParseUint: parsing "multitenant": invalid syntax`

### To Reproduce

Try to send metrics to multitenant storage at victoriametrics v1.93.2 (and newer)

### Version

```
/vmagent-prod -version
vmagent-20230901-102917-tags-v1.93.2-0-g9133a5acb1

/vminsert-prod -version
vminsert-20230901-104629-tags-v1.93.2-cluster-0-g66b31e1797
```

### Logs

`warn VictoriaMetrics/app/vminsert/main.go:213 remoteAddr: "10.10.1.161:55742"; requestURI: /insert/multitenant/prometheus/api/v1/write; auth error: cannot parse accountID from "multitenant": strconv.ParseUint: parsing "multitenant": invalid syntax`

### Used command-line flags

vmagent:
```
-promscrape.config=/etc/vmagent/vmagent_config.yml
-promscrape.config.strictParse=false
-promscrape.maxScrapeSize=200MB
-promscrape.minResponseSizeForStreamParse=60MB"
-promscrape.cluster.membersCount=6
-promscrape.cluster.memberNum=0
-promscrape.cluster.replicationFactor=1
-remoteWrite.url=http://<vminsert>:8480/insert/multitenant/prometheus/api/v1/write     
```

### Additional information

It looks like the problem is related to https://github.com/VictoriaMetrics/VictoriaMetrics/commit/1bba4c5118408eb082664db67a194950a3bd84cb

**Comment by valyala** [comment]
@chevvi , thanks for filing the bugreport! It should be fixed in the commit 7db72dd7e63fa9aff80ffb4da71a4cea060f1c76 , which will be included in the next 1.93.x LTS release. In the mean time you can build `vminsert` from this commit according to [these docs](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#building-from-sources) and verify whether `/insert/multitenant/*` urls work as intended.

The issue is related to https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4910 .

**Comment by valyala** [comment]
The issue has been fixed in [v1.93.4](https://docs.victoriametrics.com/CHANGELOG.html). Closing it as fixed.

**Comment by valyala** [comment]
The issue has been fixed also in [v1.87.9 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.87.9).
